### ISSUE TITLE
Monologue mode

### ISSUE THREAD

**Issue by jackmcbarn** [CensoredUsername/unrpyc#99]
Ren'Py supports monologue mode now: https://www.renpy.org/doc/html/dialogue.html#monologue-mode

We currently don't, so any uses of it will be turned into a bunch of individual say statements instead, which results in line numbers being wrong.

**Comment by CensoredUsername** [comment]
And it even comes along with the new "rpy monologue" statement which is actually a parser-only directive to just make things a bit more annoying.

This one's going to be jolly. Do all resulting say statements have their lineinfo set to the same line?

**Comment by jackmcbarn** [comment]
Yep. Although it doesn't all turn into say statements. It can also add an `nvl clear` user statement that way: https://www.renpy.org/doc/html/text.html#text-tag-clear

I do have an idea for this though.

**Comment by CensoredUsername** [comment]
The addition of `rpy monologue none` causes several additional complications. we'll likely have to scan all Say statement in the file, and see if any of those would normally trigger monologue mode, and if we detect that, explicitly disable it.

**Comment by madeddy** [comment]
Sry to bother about this, but could it be the code for this has already been completed? In #175 it was checked of the list and mentioned in one post from you.

I tried to find the code for it but had no success. Maybe I just don't recognize the right places.

**Comment by CensoredUsername** [comment]
Nothing's been done here. If it's used, line numbers can be wrong.

**Comment by madeddy** [comment]
Ah ok, thanks. Something to remember for the time being in case i encounter strange behavior.
